docs/documentation/maya/simple_setup.md

```diff
@@ -1,37 +1,62 @@
 # A Simple Setup
 
-This page will be focused on displaying the process of creating and setting up each deformer in maya showing the process step by step.
+This page is dedicated to explain, step by step, the process of creating and setting every Adonis deformer in Maya. The scenarios set up here will be basic cases with the minimum configuration to start observing expected results in each case.
 
 ## AdnSkin simple setup
 
-For AdnSkin we will start with a scene with the following elements:
+To create a basic scenario using the AdnSkin deformer we will start with a scene with the following elements:
```

Comment:
Please avoid the use of we wil and rewrite it in a more non-direct way. For example here:
```suggestion
To create a basic scenario using the AdnSkin deformer start with a scene with the following elements:
```

Please apply this to the whole document to make it more impersonal

Reply:
Done